src/devices/Ad7193/Ad7193.cs

```diff
@@ -0,0 +1,855 @@
+﻿using System;
```

Comment:
copyright header missing

Reply:
What is a copyright header?

Reply:
See i.e. https://github.com/dotnet/iot/blob/master/src/System.Device.Gpio/System/Device/Pwm/PwmChannel.Linux.cs#L1-L3

We got guidelines from higher up that every `.cs` file should have it